src/bootlint.js

```diff
@@ -1100,6 +1104,39 @@ var LocationIndex = _location.LocationIndex;
             reporter('`.modal-dialog` must have a `role="document"` attribute.', modalDialogs);
         }
     });
+    addLinter("E050", function requireFormControlClass($, reporter) {
+        //Select inputs of the proper type that do not have a `.form-control` class OR a `.col-*-*` class
+        var colSelector = COL_CLASSES.join(",");
+        var inputs = $('select:not(.form-control,' + colSelector + '), textarea:not(.form-control,' + colSelector + '), ' + (FORM_CONTROL_VALID_TYPES.map(function (type) {
+                return 'input:not(.form-control,' + colSelector + ')[type="' + type + '"]';
+            }).join(',')
+        ));
+        if (inputs.length) {
+            inputs.each(function (i, el) {
```

Comment:
No need for `each` here, I believe. Just pass `inputs` as the 2nd arg to `reporter`